rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LocationResource.java

```diff
@@ -166,6 +185,11 @@ public Response create(LocationSpec locationSpec) {
     @Override
     @Deprecated
     public void delete(String locationId) {
+        if (!Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.DELETE_LOCATION, Entitlements.StringAndArgument.of(locationId, "delete"))) {
+            throw WebResourceUtils.forbidden("User '%s' is not authorized to delete locations",
```

Comment:
The message should say `User '%s' is not authorized to delete the location '%s'`